### DISCUSSION TITLE
continue and taskwarrior's on-modify hook

### DISCUSSION THREAD

**Discussion by gour** [GothenburgBitFactory/timewarrior#596] [Q&A] [answered]
Hello,

I'm long time taskwarrior user, but after trying some other task-management tools, I'm back...However, this time I'd like to add Timewarrior to the table. :-)

Afaiu, starting taskwarrior will trigger (via on-modify hook) timewarrior to start  tracking specific time and completing it will stop tracking.

However, I'm interested how to achieve `start --> stop --> continue --> stop` workflow which is available within Timewarrior, but can't figure out how to do it from within Taskwarrior by using on-modify hook script?

Sincerely,
Gour

**Comment by lauft** [comment]
`continue` is just a convenience command which picks the tags for your tracking from previous ones. Those tags should be already provided by your task in Taskwarrior. So just starting your task again should "continue" time tracking for it. Or is there something I am missing? 🤔

**Comment by gour** [comment, marked as answer]
> `continue` is just a convenience command which picks the tags for your tracking from previous ones. Those tags should be already provided by your task in Taskwarrior. So just starting your task again should "continue" time tracking for it. 

Brilliant!

> Or is there something I am missing? 🤔

Nope, thank you very much for the explanation as well as for your work on Timewarrior...along with Taskwarrior such a great combination. :sunglasses:
